compliance_checker/acdd.py

```diff
@@ -69,7 +69,7 @@ def check_recommended(self, ds):
     def check_suggested(self, ds):
         return [
             'contributor_name',
-            ('contributor_role', ['principalInvestigator', 'author']),
+            'contributor_role',
```

Comment:
1.2, I didn't know 1.3 was released yet, but we should update the compliance checker to support 1.3